src/HW47.py

```diff
@@ -0,0 +1,11 @@
+# Даны два натуральных числа.
+# Вычислите их наибольший общий делитель
+# при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+num1 = int(input('Введите первое число\n'))
+num2 = int(input('Введите второе число\n'))
+while num1 and num2:
+    if num1 > num2:
+        num1 %= num2
+    else:
+        num2 %= num1
+print(num1 + num2)

```

Comment:
it's a cheat. Algorithm doesn't expect result in sum
you can rework algorithm or use ternary operator